Cargo.toml

```diff
@@ -28,8 +34,9 @@ tracing-subscriber = { version="^0.3.15", features = ["env-filter", "json"] }
 tower-http = { version = "^0.3.0", features = ["trace"] }
 axum = { version = "^0.5.16", features = ["headers"] }
 clap = { version = "^3.0.14", features = ["derive", "env"] }
-hyper = { version = "^0.14.17", features = ["full"] }
-tokio = { version = "^1.15.0", features = ["full"] }
+# hyper = { version = "^0.14.20", features = ["full"] }
+hyper = { git = "https://github.com/rjzak/hyper", branch = "wasi_wip", features = ["server"] }
+tokio = { version = "^1.21.2", features = ["full"] }
```

Comment:
Do we really need `"full"` or is only a subset of features enough to cover our use case?

Reply:
Could you file an issue to revise the list of features we enable in our dependencies and to remove redundant dependencies?

Reply:
Full was there because that was in Cargo.toml when I began working on Wasi support. I've since reduced it to `rt` and `macros`, and `rt-multi-threaded` for non-Wasi targets.
And https://github.com/profianinc/steward/issues/66